xbmc/linux/LinuxTimezone.cpp

```diff
@@ -197,13 +197,13 @@ std::string CLinuxTimezone::GetCountryByTimezone(const std::string timezone)
 
 void CLinuxTimezone::SetTimezone(std::string timezoneName)
 {
-  bool use_timezone = false;
-  
 #if !defined(TARGET_DARWIN)
-  use_timezone = true;
+  bool use_timezone = true;
 #else
   if (g_sysinfo.IsAppleTV2())
-    use_timezone = true;  
+    bool use_timezone = true;
+  else
+    bool use_timezone = false;
```

Comment:
Actually, I will remove that part. It was about not using the previous value of that variable.